libcontainer/cgroups/fs/memory_test.go

```diff
@@ -94,12 +94,17 @@ func TestMemorySetNegativeMemoryswap(t *testing.T) {
 		memoryBefore     = 314572800 // 300M
 		memoryAfter      = 524288000 // 500M
 		memoryswapBefore = 629145600 // 600M
-		memoryswapAfter  = 629145600 // 600M
+		memoryswapAfter  = 0         // Unlimited
```

Comment:
We took negative value means not change in unit test, and that's the point of this unit test, since that's not true after your PR, you can remove that unit test, and no need to add another.